### ISSUE TITLE
Add select config to `GET /store/auth/:email`

### ISSUE THREAD

**Issue by olivermrbl** [medusajs/medusa#1096]
In `GET /store/auth/:email`, replace the current check to use `customer.has_account` on [these lines](https://github.com/medusajs/medusa/blob/master/packages/medusa/src/api/routes/store/auth/exists.ts) like so: 

```
const customer = await customerService.retrieveByEmail(email)
res.status(200).json({ exists: customer.has_account })
```

**Comment by gefgu** [comment]
Hi! Can I get assigned to work on this issue?

**Comment by gaurav1999** [comment]
@olivermrbl Can we close this issue, as it's merged ?

**Comment by olivermrbl** [comment]
Indeed @gaurav1999! Closed by #1122
